modules/charon-core/src/main/java/org/wso2/charon3/core/utils/ResourceManagerUtil.java

```diff
@@ -399,15 +399,15 @@ private static void removeAttributesFromList(List<AttributeSchema> attributeSche
      */
     public static int processCount(String countStr) throws BadRequestException {
 
-        int count = 0;
-        if (countStr == null) {
+        int count;
+        if (countStr == null || countStr.isEmpty()) {
```

Comment:
What about using StringUtils.isBlank() or StringUtils.isEmpty?

https://commons.apache.org/proper/commons-lang/javadocs/api-2.6/org/apache/commons/lang/StringUtils.html#isBlank(java.lang.String)

https://commons.apache.org/proper/commons-lang/javadocs/api-2.6/org/apache/commons/lang/StringUtils.html#isEmpty(java.lang.String)

Reply:
@denuwanthi then we'll have to add that dependency as well, right?